android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -188,13 +188,15 @@ public void setLargeIcon(Object icon)
 		setProperty(TiC.PROPERTY_LARGE_ICON, icon);
 	}
     
-    	@Kroll.method @Kroll.setProperty
-    	public void setColor(String color)
-    	{
-        	notificationBuilder.setColor(TiColorHelper.parseColor(color));
-        	setProperty(TiC.PROPERTY_COLOR, color);
-    	}
-
+    @Kroll.method @Kroll.setProperty
+    public void setColor(String color)
+    {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)) {
+            notificationBuilder.setColor(TiColorHelper.parseColor(color));
+        }
+        setProperty(TiC.PROPERTY_COLOR, color);
+    }
+    
 	@Kroll.method @Kroll.setProperty
```

Comment:
Please check the indentation. I fixed it previously but you want to check this as well - maybe for future PR's 😙.